Fixed typos (#88)
epicweb-dev/react-suspense: exercises/01.fetching/01.problem.throw/README.mdx

which retrieves the data from `./shared/ship-api-utils.server.ts`.

The problem is that while the `fetch` request is ongoing, the user is just

> staring at a blank white screen. Now we could for sure improve our `index.tsx`

Reviewer: This should do it.

```suggestion
staring at a blank white screen. Now we could for sure improve the HTML document
```
